package.json

```diff
@@ -23,9 +23,7 @@
     "moment": "^2.10.6",
     "numeral": "^1.5.3",
     "react-addons-shallow-compare": "^0.14.7",
-    "recompose": "^0.15.1"
-  },
-  "peerDependencies": {
+    "recompose": "^0.15.1",
     "react": "^0.14.2",
```

Comment:
`react` and `react-dom` should actually stay in peerDependencies - otherwise Reactochart might install and use its own version of React when included in a project, which is bad and breaks React. Will update this in my batch of changes coming soon